src/net_processing.cpp

```diff
@@ -4452,13 +4465,15 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
             LOCK(cs_main);
 
             bool expected_blocktxn = false;
+            bool first_in_flight = true;
             auto range_flight = mapBlocksInFlight.equal_range(resp.blockhash);
             while (range_flight.first != range_flight.second) {
                 auto [node_id, block_it] = range_flight.first->second;
                 if (node_id == pfrom.GetId() && block_it->partialBlock) {
                     expected_blocktxn = true;
                     break;
                 }
+                first_in_flight = false;
```

Comment:
This logic is different to the previous `first_in_flight`; probably should be the same?

Reply:
done, cleaner thank you